File: 1074.cpp
#include <iostream>
using namespace std;

int main(){
    int N, o, w, vezes;
    cin >> N;
    string lista[N];
    for(int i = 0; i < N; i++){
        cin >> o;
        if(o < 0){
            if(o % 2 != 0){
                lista[vezes] = "ODD NEGATIVE";
            }
            if(o % 2 == 0){
                lista[vezes] = "EVEN NEGATIVE";
            }
        }
        if(o == 0){
            lista[vezes] = "NULL";
        }
        if(o > 0){
            if(o % 2 != 0){
                lista[vezes] = "ODD POSITIVE";
            }
            if(o % 2 == 0){
                lista[vezes] = "EVEN POSITIVE";
            }
        }
        vezes++;
    }
    for(w = 0; w < N; w++){
        cout << lista[w] << "\n";
    }
}

File: 1074.java
import java.util.Scanner;
public class Main {
public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int N = input.nextInt();
        for(int i = 0; i < N; i++) {
                int o = input.nextInt();
                if(o < 0) {
                        if(o % 2 != 0) {
                                System.out.println("ODD NEGATIVE");
                        }
                        if(o % 2 == 0) {
                                System.out.println("EVEN NEGATIVE");
                        }
                }
                if(o == 0) {
                        System.out.println("NULL");
                }
                if(o > 0) {
                        if(o % 2 != 0) {
                                System.out.println("ODD POSITIVE");
                        }
                        if(o % 2 == 0) {
                                System.out.println("EVEN POSITIVE");
                        }
                }
        }
}
}
